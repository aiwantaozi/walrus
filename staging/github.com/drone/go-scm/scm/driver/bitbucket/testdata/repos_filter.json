{
  "pagelen": 1,
  "values": [
    {
      "scm": "git",
      "website": "",
      "has_wiki": false,
      "uuid": "{7dd600e6-0d9c-4801-b967-cb4cc17359ff}",
      "links": {
        "watchers": {
          "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin1\/watchers"
        },
        "branches": {
          "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin1\/refs\/branches"
        },
        "tags": {
          "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin1\/refs\/tags"
        },
        "commits": {
          "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin1\/commits"
        },
        "clone": [
          {
            "href": "https:\/\/brydzewski@bitbucket.org\/atlassian\/stash-example-plugin1.git",
            "name": "https"
          },
          {
            "href": "git@bitbucket.org:atlassian\/stash-example-plugin1.git",
            "name": "ssh"
          }
        ],
        "self": {
          "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin1"
        },
        "source": {
          "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin1\/src"
        },
        "html": {
          "href": "https:\/\/bitbucket.org\/atlassian\/stash-example-plugin1"
        },
        "avatar": {
          "href": "https:\/\/bytebucket.org\/ravatar\/%7B7dd600e6-0d9c-4801-b967-cb4cc17359ff%7D?ts=default"
        },
        "hooks": {
          "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin1\/hooks"
        },
        "forks": {
          "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin1\/forks"
        },
        "downloads": {
          "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin1\/downloads"
        },
        "pullrequests": {
          "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin1\/pullrequests"
        }
      },
      "fork_policy": "allow_forks",
      "name": "stash-example-plugin1",
      "project": {
        "key": "PROJ",
        "type": "project",
        "uuid": "{8b56daff-dbc7-4cae-a7a3-1228c526906b}",
        "links": {
          "self": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/teams\/atlassian\/projects\/PROJ"
          },
          "html": {
            "href": "https:\/\/bitbucket.org\/account\/user\/atlassian\/projects\/PROJ"
          },
          "avatar": {
            "href": "https:\/\/bitbucket.org\/account\/user\/atlassian\/projects\/PROJ\/avatar\/32"
          }
        },
        "name": "Project: Atlassian"
      },
      "language": "",
      "created_on": "2013-04-15T03:05:05.595458+00:00",
      "mainbranch": {
        "type": "branch",
        "name": "master"
      },
      "full_name": "atlassian\/stash-example-plugin1",
      "has_issues": false,
      "owner": {
        "username": "atlassian",
        "display_name": "Atlassian",
        "type": "team",
        "uuid": "{02b941e3-cfaa-40f9-9a58-cec53e20bdc3}",
        "links": {
          "self": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/teams\/atlassian"
          },
          "html": {
            "href": "https:\/\/bitbucket.org\/atlassian\/"
          },
          "avatar": {
            "href": "https:\/\/bitbucket.org\/account\/atlassian\/avatar\/32\/"
          }
        }
      },
      "updated_on": "2018-04-01T16:36:35.970175+00:00",
      "size": 1116345,
      "type": "repository",
      "slug": "stash-example-plugin1",
      "is_private": true,
      "description": "Examples on how to decorate various pages around Stash."
    }
  ],
  "next": "https:\/\/api.bitbucket.org\/2.0\/repositories?pagelen=1&after=PLACEHOLDER&role=member"
}