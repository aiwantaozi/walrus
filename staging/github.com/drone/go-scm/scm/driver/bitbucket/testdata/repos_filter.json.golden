[
    {
        "ID": "{7dd600e6-0d9c-4801-b967-cb4cc17359ff}",
        "Namespace": "atlassian",
        "Name": "stash-example-plugin1",
        "Perm": null,
        "Branch": "master",
        "Private": true,
        "Clone": "https://bitbucket.org/atlassian/stash-example-plugin1.git",
        "CloneSSH": "git@bitbucket.org:atlassian/stash-example-plugin1.git",
        "Link": "https://bitbucket.org/atlassian/stash-example-plugin1",
        "Created": "2013-04-15T03:05:05.595458Z",
        "Updated": "2018-04-01T16:36:35.970175Z"
    }
]